src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.dao.FruitDao;
+import core.basesyntax.model.dao.FruitDaoImpl;
+import core.basesyntax.model.dto.Fruit;
+import core.basesyntax.service.interfaces.FruitService;
+import core.basesyntax.service.interfaces.OperationHandler;
+import core.basesyntax.storage.FruitStorage;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FruitServiceImplTest {
+    private static final FruitDao fruitDao = new FruitDaoImpl();
+    private static final Map<Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+    private static final List<String> VALID_DATA =
```

Comment:
```suggestion
    private static final List<String> validData =
```
List is not deeply immutable so we need ot use regular naming

Reply:
Didn't know about this naming rule, thanks!